tar/util.c

```diff
@@ -140,6 +140,7 @@ safe_fprintf(FILE *f, const char *fmt, ...)
 		} else {
 			/* Leave fmtbuff pointing to the truncated
 			 * string in fmtbuff_stack. */
+			fmtbuff = fmtbuff_stack;
```

Comment:
It's tricky to spot, but once you see it, it's kind of obvious -- Coverity was just a tad bit obfuscated about this case (it took me a bit to spot it before.. heh).